### ISSUE TITLE
Post/Pages: User can tap on multiple cell buttons simultaneously

### ISSUE THREAD

**Issue by diegoreymendez** [wordpress-mobile/WordPress-iOS#3894]
Right now its possible to tap on multiple cell buttons simultaneously (such as "Edit" + "View", or "Edit" + "Trash"), and both actions are performed, causing a weird experience on the user side.

Nothing seems to break too badly though, at least in my tests.

**Comment by aerych** [comment]
I'm having trouble reproducing this one on a device, and can't at all on the simulator.  Is there a trick to where you need to tap?  If you have a video example that could be very helpful.

**Comment by diegoreymendez** [comment]
Hey Eric,

So [here's a video](https://cloudup.com/coUO9bK4xjW) showing how to reproduce this on the simulator.

I actually got this issue first on the physical device by doing a very quick simultaneous tap on the edit and trash buttons.  It's not that easy though, as the device seems to sometimes just register one tap.  The easiest way I found is to use the index and middle finger from the same hand, by lowering your entire hand towards the device when you want to tap (makes sense?).  [Here's the video](https://cloudup.com/cVFUwVCUYmw) showing the issue on a physical device.
